python/spear/game_world_service.py

```diff
@@ -2,9 +2,92 @@
 # Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
 #
 
+from enum import Enum
 import json
 
 class GameWorldService():
+
+    class PropertyDesc():
+        property_
+        value_ptr_
+
+    # should match the EIncludeSuperFlag::Type enum in Engine\Source\Runtime\CoreUObject\Public\UObject\Class.h
```

Comment:
forward slashes for consistency

Reply:
See my most recent design sketch to avoid duplicating most of these enums in Python.